### PR TITLE
Improve the readme

### PR DESCRIPTION
**Pull request by axtloss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,17 @@ A neumorphic take on adwaita theme
...
-Clone the repository inside `~/.local/share/icons` and select the theme in gnome-tweaks
+Clone the repository into `~/.local/share/icons/Neuwaita`:
+```
+mkdir -p ~/.local/share/icons
+git clone https://github.com/Ashwatthaamaa/Neuwaita.git ~/.local/share/icons/Neuwaita
+```
 
-## Requesting for new icons :
-I understand you really want the icon but while making an icon request please mention actual name of the icon file that you want to request.
+## Requesting new icons :
```
Comment (TheEvilSkeleton) [inline README.md]:
```suggestion
## Requesting new icons:
```

File: README.md [hunk 2]
```diff
@@ -6,13 +6,17 @@ A neumorphic take on adwaita theme
...
+I understand you really want the icon but while making an icon request please mention the [actual name of the icon](https://github.com/Ashwatthaamaa/Neuwaita#how-to-find-the-actual-name-of-the-icon-) file that you want to request.
 
-## How to find the **Actual name** of the icon ?
-* Icons are usually located inside `/usr/share/icons/hicolor/scalable/<Actual name of your app>`
-* For Flatpaks the location is `/var/lib/flatpak/app/<Actual name of your app>` the name of the folder is the name of your icon
+### How to find the **Actual name** of the icon ?
```
Comment (TheEvilSkeleton) [inline README.md]:
```suggestion
### How to find the **Actual name** of the icon?
```

File: README.md [hunk 3]
```diff
@@ -6,13 +6,17 @@ A neumorphic take on adwaita theme
...
+Clone the repository into `~/.local/share/icons/Neuwaita`:
+```
+mkdir -p ~/.local/share/icons
+git clone https://github.com/Ashwatthaamaa/Neuwaita.git ~/.local/share/icons/Neuwaita
+```
 
-## Requesting for new icons :
-I understand you really want the icon but while making an icon request please mention actual name of the icon file that you want to request.
+## Requesting new icons :
+I understand you really want the icon but while making an icon request please mention the [actual name of the icon](https://github.com/Ashwatthaamaa/Neuwaita#how-to-find-the-actual-name-of-the-icon-) file that you want to request.
```
Comment (TheEvilSkeleton) [inline README.md]:
```suggestion
I understand you really want the icon but while making an icon request please mention the [actual name of the icon](#how-to-find-the-actual-name-of-the-icon) file that you want to request.
```

File: README.md [hunk 4]
```diff
@@ -6,13 +6,25 @@ A neumorphic take on adwaita theme
...
+Clone the repository into `/usr/share/icons`
+```
+sudo mkdir -p /usr/share/icons/Neuwaita
+git clone https://github.com/Ashwatthaamaa/Neuwaita.git /usr/share/icons/Neuwaita
+```
 
-## Requesting for new icons :
-I understand you really want the icon but while making an icon request please mention actual name of the icon file that you want to request.
+## Requesting new icons :
+I understand you really want the icon but when making an icon request please include the [actual name of the icon](https://github.com/Ashwatthaamaa/Neuwaita#how-to-find-the-actual-name-of-the-icon-) that you want to request.
```
Comment (TheEvilSkeleton) [inline README.md]:
```suggestion
I understand you really want the icon but when making an icon request please include the [actual name of the icon](#how-to-find-the-actual-name-of-the-icon) that you want to request.
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

**Review by meisme-dev — CHANGES_REQUESTED** [review]
"A neumorphic take on adwaita theme" should be "A neumorphic take on the Adwaita theme"

**Comment by axtloss** [comment]
Should be ready to merge now.

**Comment by TheEvilSkeleton** [comment]
@Ashwatthaamaa a little tip, you might want to squash commits before merging. This prevents the commit history from cluttering:
image

**Pull request merged**